apps/docs/next.config.mjs

```diff
@@ -149,6 +149,14 @@ const nextConfig = {
         basePath: false,
         permanent: false,
       },
+
+      // Redirect Dart Edge docs to Invertase's docs
+      {
+        source: '/docs/guides/functions/dart-edge',
+        destination: 'https://invertase.docs.page/dart_edge/platform/supabase',
```

Comment:
For future-forward security, I'm not a huge fan of redirecting to third parties unless they are super huge players that we expect to last forever (like AWS 😄), I'm happy enough to let this 404 or just redirect to functions index page

Reply:
Sorry, this PR slipped through the crack. Sounds good. Let's do that 👍